### DISCUSSION TITLE
Llama.cpp server stopping

### DISCUSSION THREAD

**Discussion by appvoid** [ggml-org/llama.cpp#3513] [Q&A]
I'm trying to stop ./server inference on the middle of the process without any good results so far.

I'm stuck here. Can someone explain me better how can I "stop" the the process when requesting a streamed inference on the server?

I'm using javascript's Fetch.

**Comment by jhen0409** [comment]
The server example use fetch & AbortController, it should work in the token generating. However the prompt processing is usually one-run, so it may not be interrupted immediately unless n_batch is set to a small value.
